Frends.SFTP.ReadFile/Frends.SFTP.ReadFile/Frends.SFTP.ReadFile.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+	  <TargetFramework>net6.0</TargetFramework>
+	  <ImplicitUsings>enable</ImplicitUsings>
+	  <IncludeSource>true</IncludeSource>
+	  <AssemblyName>Frends.SFTP.ReadFile</AssemblyName>
+	  <RootNamespace>Frends.SFTP.ReadFile</RootNamespace>
+
+	  <Version>1.0.0</Version>
+	  <Authors>Frends</Authors>
+	  <Copyright>Frends</Copyright>
+	  <Company>Frends</Company>
+	  <Product>Frends</Product>
+	  <PackageTags>Frends</PackageTags>
+	  <PackageLicenseExpression>MIT</PackageLicenseExpression>
+	  <GenerateDocumentationFile>true</GenerateDocumentationFile>
+	  <Description>Executes file transfer through SFTP connection.</Description>
```

Comment:
I think this should be something like read file or get file from SFTP server or something.

Reply:
Fixed with "Read file through SFTP connection."